src/ed25519-donna/ed25519-donna-portable.h

```diff
@@ -20,6 +20,9 @@
 	#include <sys/param.h>
 	#define DONNA_INLINE inline __attribute__((always_inline))
 	#define DONNA_NOINLINE __attribute__((noinline))
+        #ifdef ALIGN
+          #undef ALIGN
+        #endif
```

Comment:
Do we really need to check if it's defined before the undef? Why not just `#undef ALIGN`?

Reply:
agree - will fix.